src/components/DataRoom/FeesEarned/styles.module.css

```
@@ -0,0 +1,98 @@
+.sectionContainer {
+  height: 200vh;
+  display: flex;
+}
+
+.stickyContainer {
+  position: sticky;
+  top: 0;
+  width: 100%;
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: end;
```

```suggestion
  align-items: flex-end;
```

please pay attention to the property name because the linter is complaining